DriverAchievementUI/src/main/java/com/drivequant/drivekit/driverachievement/ui/streaks/fragment/StreaksListFragment.kt

```diff
@@ -0,0 +1,94 @@
+package com.drivequant.drivekit.driverachievement.ui.streaks.fragment
+
+import android.arch.lifecycle.Observer
+import android.arch.lifecycle.ViewModelProviders
+import android.os.Bundle
+import android.support.v4.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import com.drivequant.drivekit.driverachievement.streak.AchievementSyncStatus
+
+import com.drivequant.drivekit.driverachievement.ui.StreaksViewConfig
+import com.drivequant.drivekit.driverachievement.ui.streaks.viewmodel.StreaksListViewModel
+import kotlinx.android.synthetic.main.fragment_streaks_list.*
+import android.support.v7.widget.LinearLayoutManager
+import com.drivequant.drivekit.driverachievement.ui.R
+import com.drivequant.drivekit.driverachievement.ui.streaks.adapter.StreaksListAdapter
+
+
+class StreaksListFragment : Fragment() {
+
+    private lateinit var listViewModel: StreaksListViewModel
+    private lateinit var streaksViewConfig: StreaksViewConfig
+    private var listAdapter: StreaksListAdapter? = null
```

Comment:
Rendre listAdapter non-nullable grâce à `lateinit` puis vérifier son initialisation de la suite de la façon suivante : `this::listAdapter.isInitialized`

Reply:
Done